tensilelite/Tensile/Contractions.py

```diff
@@ -498,6 +498,9 @@ def CompoundPredicates(cls, state, problemType):
             value = globalParameters['MinKForGSU']
             rv += [cls('GlobalSplitUCheckMinK', value=[value, state["GlobalSplitU"]])]
 
+        if ('WorkGroupMappingXCC' in state) and ('WorkGroupMappingXCCGroup' in state) and (state['WorkGroupMappingXCCGroup'] == -1):
```

Comment:
WorkgroupMappingXCCCheck is only necessary when WorkGroupMappingXCCGroup  is -1?